components/nexus-core/src/main/java/org/sonatype/nexus/internal/httpclient/SharedHttpClientConnectionManager.java

```diff
@@ -59,7 +61,9 @@ public SharedHttpClientConnectionManager(
       final List<SSLContextSelector> sslContextSelectors,
       @Named("${nexus.httpclient.connectionpool.size:-20}") final int connectionPoolSize,
       @Named("${nexus.httpclient.connectionpool.maxSize:-200}") final int connectionPoolMaxSize,
-      @Named("${nexus.httpclient.connectionpool.idleTime:-30s}") final Time connectionPoolIdleTime)
+      @Named("${nexus.httpclient.connectionpool.idleTime:-30s}") final Time connectionPoolIdleTime,
+      @Named("${nexus.httpclient.connectionpool.evictingDelayTime:-5s}") final Time connectionPoolEvictingDelayTime,
+      @Named("${nexus.httpclient.connectionpool.validateAfterInactivityTime:-2s}") final Time connectionPoolValidateAfterInactivityTime)
```

Comment:
@mcculls that's the default value in PoolingHttpClientConnectionManager ctor, that we later disable by setting it to -1. I think this new 4.4 feature should not be kept disabled.